packages/xod-project/src/messages.js

```diff
@@ -4,4 +4,16 @@ export const formatDeadReferencesFound = (patchPath, submessage) => [
   'Fix or delete them to continue.',
 ].join('\n');
 
-export default {};
+// Variadics
+
+export const patchHasNoVariadicMarkers = patchPath => `Patch "${patchPath}" has no variadic markers.`;
+
+export const patchHasMoreThanOneVariadicMarkers = patchPath => `Patch "${patchPath}" has more than one variadic-* marker`;
```

Comment:
Not sure translatingTheFullErrorSentenceInAjsSymbolName is best idea. Isn’t it better?

```
noVariadicMarkers
tooManyVariadicMarkers
notEnoughVariadicInputs
```